.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ commands:
   setup-rust-min-version:
     steps:
       # Technically should be on 1.47 here, but that boat sailed - 1.50 has been our effective min for some time now.
-      - run: rustup override set 1.50.0
+      - run: rustup override set 1.53.0
```

Comment:
See https://searchfox.org/mozilla-central/source/python/mozboot/mozboot/util.py#20 - this should stay at 1.53.0 (so the only change here should be the removal of the comment)

Reply:
(oops - sorry - this comment is confused and wrong! The change is correct, but please remove the comment)

Reply:
hah figured what you meant! Removed the comment, I did add the you posted above though, helpful context